core/src/main/java/hudson/util/XStream2.java

```diff
@@ -415,7 +415,7 @@ public Object unmarshal(HierarchicalStreamReader reader, UnmarshallingContext co
 
         private PluginManager pm;
 
-        @SuppressWarnings("NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE") // classOwnership checked for null so why does FB complain?
+        @SuppressFBWarnings("NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE") // classOwnership checked for null so why does FB complain?
```

Comment:
Good practice is to add a clarification of suppression reasons. There is a field in the attribute for it